package.json

```diff
@@ -629,6 +629,7 @@
                             "enum": [
```

Comment:
Could you by the way add `"enumDescriptions"` for this setting here? 😄

Reply:
Fixed, and optimize star logic.